Modules/Devices/src/Devices.Application/Identities/Commands/StartDeletionProcessAsUser/Handler.cs

```diff
@@ -1,14 +1,12 @@
 using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
-using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.EventBus;
 using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
 using Backbone.DevelopmentKit.Identity.Entities;
 using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
-using Backbone.Modules.Devices.Application.IntegrationEvents.Outgoing;
 using MediatR;
 
-namespace Backbone.Modules.Devices.Application.Identities.Commands.StartDeletionProcess;
+namespace Backbone.Modules.Devices.Application.Identities.Commands.StartDeletionProcessAsUser;
 
-public class Handler : IRequestHandler<StartDeletionProcessCommand, StartDeletionProcessResponse>
+public class Handler : IRequestHandler<StartDeletionProcessAsUserCommand, StartDeletionProcessAsUserResponse>
```

Comment:
Damn. It should be "as owner", not "as user". But you could have guessed this yourself. :P